pdns/recursordist/test-syncres_cc.cc

```diff
@@ -558,3 +558,7 @@ void pushAlmostExpiredTask(const DNSName& qname, uint16_t qtype, time_t deadline
 {
   g_test_tasks.push({qname, qtype, deadline, true, nullptr});
 }
+
+void pushResolveTask(const DNSName& qname, uint16_t qtype, time_t now, time_t deadline)
```

Comment:
```suggestion
static void pushResolveTask(const DNSName& qname, uint16_t qtype, time_t now, time_t deadline)
```

Reply:
That wont fly as is declared non-static in `rec-taskqueue.hh` (and it needs to be non-static).